ztp/source-crs/ClusterImageSet.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: hive.openshift.io/v1
```

Comment:
The source-cr directory contains CRs that get applied to the spoke clusters. Is there a reason you would need a ClusterImageSet on the spoke (typically we use it only on the hub cluster for installation)?

Reply:
Not really, is just to automate the ClusterImageSet creation. I will delete it